src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThread.scss

```diff
@@ -1,7 +1,23 @@
-@import '../../../../styles/variables';
+@import '../../../common/variables';
 
 .AnnotationThread {
-    max-width: 340px;
-    max-height: 600px;
+    min-width: 340px;
     background-color: $white;
+
+    .bcs-ActivityThread {
+        position: relative;
+        max-height: 528px;
```

Comment:
Is this being derived from something? Is it `600px` minus some other value(s)? If so, we should have those values as variables, ideally.

Reply:
I don't think it's based on some other value.

As discussed, I've added `.bcs-ActivityThread-content` wrapper in the `ActivityThread`, so that the form is not used outside of it.